src/Microsoft.OData.Core/ODataBinaryStreamReader.cs

```diff
@@ -50,7 +50,7 @@ internal ODataBinaryStreamReader(Func<char[], int, int, int> reader)
             Debug.Assert(reader != null, "reader cannot be null");
 
             this.reader = reader;
-            chars = new char[charLength];
+            this.chars = new char[this.charLength];

```

Comment:
Note: Changes to this file are primarily intended to make it easy to tell apart class and local variables